expertise/create_dataset.py

```diff
@@ -15,10 +15,27 @@
 from tqdm import tqdm
 from concurrent.futures import ThreadPoolExecutor
 from collections import defaultdict
+from expertise.service.utils import mock_client
 
 class OpenReviewExpertise(object):
-    def __init__(self, openreview_client, config):
+    def __init__(self, openreview_client, config, baseurl_v2=None):
```

Comment:
Why don't you pass the openreview_client_v2 as a parameter here?

Reply:
I had originally done that but thought that too many things needed to be changed - but I can pass a V2 client instead since the V1 client is being passed in already